backend/alembic/versions/435a8e31682e_init_bookmarks.py

```diff
@@ -0,0 +1,45 @@
+"""empty message
+
+Revision ID: 435a8e31682e
+Revises: 8ccd31a4f116
+Create Date: 2023-10-23 18:44:22.900953+02:00
+
+"""
+import fastapi_users_db_sqlalchemy.generics  # noqa
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "435a8e31682e"
+down_revision = "8ccd31a4f116"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "bookmarks",
+        sa.Column("id", fastapi_users_db_sqlalchemy.generics.GUID(), nullable=False),
+        sa.Column("user", sa.Uuid(), nullable=False),
```

Comment:
Should this not be a foreign key with on delete cascade to the user so bookmarks are removed when the user record is removed?

Reply:
Good point!